.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,109 @@
+name: Bug Report
+description: File a Bug report
+title: '[BUG] '
+labels: bug
+#assignees: 'anonymous@temp'
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out forms as cleanly as possible.
+  - type: dropdown
+    id: version
+    attributes:
+      label: Dosbox-Staging Version
+      description: What version are you using? Run `dosbox --version` or type `ver` command inside DOSBox shell; If the version is reported as `git` (meaning you compiled it yourself), use `git describe` command.
+      multiple: false
+      options:
+        - 0.78.0-alpha
+        - 0.77.1
+        - 0.77.0
+        - 0.76.0
+        - 0.75.2
+        - 0.75.1
+        - 0.75.0
+        - Git
+        - Other
+    validations:
+      required: true
+  - type: dropdown
+    id: OS
+    attributes:
+      label: What Operating System are you using?
+      multiple: false
+      options:
+        - Windows 11
+        - Windows 10
+        - Windows 8
+        - Windows 7
+        - Windows Other
+        - Mac OS
+        - Linux x86
+        - Linux x86_64
+        - Linux ARM (Raspberry Pi)
+        - Other
+  - type: input
+    id: other
+    attributes:
+      label: If Other OS, please describe
+      description: Other details
+      placeholder: "Windows, Mac OSX version, Debian, Ubuntu, Arch, etc."
+    validations:
+      required: false
+  - type: input
+    id: hardware
+    attributes:
+      label: Relevant hardware info
+      description: Hardware
+      placeholder: "CPU, GPU, device brand/model: e.g. Raspberry Pi3B+, Nvidia, etc"
+    validations:
+      required: true
+  - type: checkboxes
+    id: checked
+    attributes:
+      label: Have you checked that no other similar issue already exists?
+      description: Searched issues before creating report?
+      options:
+        - label: I have searched and not found similar issues.
+          required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: A clear and concise description of what the bug is.
+      description: Describe what happens, what software were you running?
+      placeholder: "How & When does this occur?"
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce the behaviour.
+      description: How can we reproduce this?
+      placeholder: "Explain how to reproduce"
+    validations:
+      required: false
+  - type: textarea
+    id: config_file
+    attributes:
+      label: Share the config file(s) you've been using to run the program.
+      description: (dosbox-staging.conf) Please avoid pasting the full config, use attachments or links
+      placeholder: "example: texture_renderer = opengl"
```

Comment:
```suggestion
      placeholder: "example: texture_renderer = opengl"
      render: ini # syntax highlighting
```

Reply:
render:
not supported by input:
only supported in textarea:

Reply:
https://github.com/arrowgent/dosbox-staging/issues/12

example

Reply:
isn't this textarea?

Reply:
sorry, wasnt sure how to integrate it with my changes & commits.

you can of course use my fork 

https://github.com/arrowgent/dosbox-staging/tree/issueforms